File: WebAPI/WebAPI.Models/Models/Models/Team.cs
using System;

namespace WebAPI.Models.Models.Models
{
    public class Team
    {
        public Guid TeamId { get; set; }
        
        public Guid ProjectId { get; set; }
        
        public string TeamName { get; set; }
        
        public string Location { get; set; }

        public DateTime CreationDate { get; set; }
        
        public virtual int MembersCount { get; set; }
    }
}

File: web/src/mappers/team.ts
import { ITeam, ITeamSimpleModel } from '../types/team';

export const mapToTeamModel = (data): ITeam => ({
    teamId: data.teamId,
    teamName: data.teamName,
    membersCount: data.membersCount,
    projectId: data.projectId,
    location: data.location,
    users: data.users,
    creationDate: new Date(data.creationDate),
});

export const mapToSimpleTeamModel = (data): ITeamSimpleModel => ({
    teamId: data.teamId,
    teamName: data.teamName,
    location: data.location,
});
